src/engine/enginemaster.cpp

```diff
@@ -260,47 +266,78 @@ void EngineMaster::processChannels(unsigned int* busChannelConnectionFlags,
             continue;
         }
 
-        bool needsProcessing = false;
```

Comment:
Removing this flag means we'll process every channel that ```isActive()```. Previously we saved some CPU time by skipping channels that were active but did not need processing. Why remove it?